pkg/vss/vehicle-structs.go

```diff
@@ -58,6 +58,8 @@ const (
 	FieldPowertrainTractionBatteryChargingChargeLimit = "powertrainTractionBatteryChargingChargeLimit"
 	// FieldPowertrainTractionBatteryChargingIsCharging True if charging is ongoing. Charging is considered to be ongoing if energy is flowing from charger to vehicle.
 	FieldPowertrainTractionBatteryChargingIsCharging = "powertrainTractionBatteryChargingIsCharging"
+	// FieldPowertrainTractionBatteryCurrentPower Current electrical energy flowing in/out of battery. Positive = Energy flowing in to battery, e.g. during charging. Negative = Energy flowing out of battery, e.g. during driving.
+	FieldPowertrainTractionBatteryCurrentPower = "powertrainTractionBatteryCurrentPower"
```

Comment:
do we want to take this opportunity to also populate `powertrainTractionBatteryChargingIsCharging` if we're seeing a positive value here?

Reply:
I worry that cross-talk between the fields will not fit into the code generation.

Reply:
The codegen should handle multiple signals being populated from the same data point

Reply:
Speaking lazily: we only get this for Tesla, and we're for sure populating `charging` for those.